app/controllers/ems_cloud_controller.rb

```diff
@@ -160,7 +160,7 @@ def ems_cloud_form_fields
                      :provider_id                     => @ems.provider_id ? @ems.provider_id : "",
                      :hostname                        => @ems.hostname,
                      :api_port                        => @ems.port,
-                     :api_version                     => @ems.api_version,
+                     :api_version                     => @ems.api_version ? @ems.api_version : "v2",
```

Comment:
I think UI code would need to change quite a bit. Now for the new Manager, they don't have here the correct class, just regular CloudManager. So I cannot put the default value to the correct class, which is the Openstack::CloudManager. 

Although with the angular form, I think it is not supported to have default value per provider, since it doesn't do any requests when changing the provider type, the logic is in JS

So not sure, should I put the default value on ManageIQ::Providers::CloudManager? It doesn't belong there, it doesn't belong here, hard choice. :-)